test/test_functionalization_of_rng_ops.py

```diff
@@ -0,0 +1,289 @@
+# Owner(s): ["module: functorch"]
```

Comment:
Why is this functorch?

Reply:
Because I want the owner to be AOTAutograd/Dispatcher, and `test_aotdispatch.py` has same owner.

Any better suggestion?

Reply:
I would say something related to pt2. This means that when this test fail, we add the functorch module to it. But this code doesn't seem to be functorch code (even though it lives in that folder).
Maybe @zou3519 has opinion on this?

Reply:
The term "functorch" has been overloaded to mean (1) the torch.func transforms and (2) Things we created that were inspired by JAX that are now a part of our compilation workstream, like AOTAutograd. The low-level functional RNG APIs belongs in the latter.

I'm fine if you want the owner to be "module: functorch", but some other module suggestions are "module: aotdispatch" or "oncall: pt2" (and feel free to change the owner of test_aotdispatch.py to one of those)